frontend/src/Pages/AddWordPage.tsx

```diff
@@ -0,0 +1,28 @@
+import ButtonAddWord from "../components/common/ButtonAddWord";
+import InputFieldAddWord from "../components/common/InputFieldAddWord";
+import Navbar from "../components/common/Navbar";
+
+const AddWordPage = () => {
+  return (
+    <>
+      <Navbar title="E-Learning System | Admin" />
+      <div className="ml-[15rem] mt-20 flex justify-evenly w-[64rem] h-[30rem] mx-auto bg-white rounded-xl shadow-md ">
+        <div>
+          <h1 className="ml-[43px] mt-[40px] ">Add Word</h1>
+          <h3 className="ml-[47px] mt-[10px] ">Word</h3>
+          <InputFieldAddWord id="question" name="question" />
+        </div>
+        <div className="ml-[5rem]">
+          <h3 className="  mt-[60px]  ml-[47px]">Choices</h3>
+          <InputFieldAddWord id="options" name="options" />
```

Comment:
I think it is better if  we add a radio button beside on each input fields to determine the correct answer.